### PR TITLE
feat(suite): UI uplift

### LINKED ISSUE #4067: Add account: Improvements
- [x] `Activate more coins `font doesn't seem to be correct (weight)
- [x] Plus sign is not vertically alligned with `Activate more coins`  
- [ ] Improve animation from the `activate more coins` button

Screen Shot 2021-07-22 at 15 53 06 

***  

- [x] The boarder of the coin seems thinner than designed (should be 1.5px) for both states (not/selected) 

Screen Shot 2021-07-22 at 15 57 04

Screen Shot 2021-07-22 at 15 56 44 

*** 

- [ ] The label `activate more coins` underline indicate there should be a tooltip

Screen Shot 2021-07-22 at 15 58 45 

- [x] Tesnet explanation in tooltip (underline)

Screen Shot 2021-07-22 at 16 20 54

- [ ] Supported tokens tooltip needs to link to the article in Suite Guide

- [x] Too much spacing (to be discussed)

Screen Shot 2021-07-22 at 16 15 45 

- [x] Hover on "X" to close modal 

- [x] Remove these label below the account type

Screen Shot 2021-07-22 at 16 28 57

- [x] Remove account type from the coin bubble 

Screen Shot 2021-07-22 at 16 30 24

- [x] Labels not vertically aligned

Screen Shot 2021-07-22 at 16 31 48
 
 

- [ ] Remove plus sign from button 

Screen Shot 2021-07-22 at 16 33 28

- [ ] Get rid of the "TEST COIN" description

Screen Shot 2021-07-22 at 16 34 13
 

- [ ] Labels (select coin, account type) have different font weight

Screen Shot 2021-07-22 at 16 36 51

- [ ] Disable coin bubble when empty account already created (as oppose to telling the user in 2 steps by disabling the primary button) › Propose use the accounts that can be activated (= those that can have the account added)

### LINKED ISSUE #4111: ReconnectDevicePrompt__Overlay on top of the firmware update modal obstructs the close button
The ReconnectDevicePrompt__Overlay on top of the firmware update modal obstructs the close button while it's still possible to exit via pressing the Esc key - user should be able to exit the update flow any time, compared to Onboarding.

image

Suite: web 21.8.1 8dfc3fbf
Browser: Chromium 92.0.4515.107
OS: NixOS 21.05
Device: model One 1.10.1

_Originally posted by @sorooris in https://github.com/trezor/trezor-suite/issues/3718#issuecomment-893390160_

### LINKED ISSUE #5162: Tooltips for navigation actions (menu items)
add Tooltips for navigation actions: 

- [ ] Discrete mode
- [ ] Custom backends
- [ ] Tor
- [ ] Early Access Program
- [ ] Notifications
- [ ] Settings

Maybe it would need some bigger refactoring for Tooltips, because I wasn't able to find a fix for this ugly behavior on click in reasonable time:

https://user-images.githubusercontent.com/3729633/159301093-db009d33-a6a8-4ca0-9f32-1ec14e05aa17.mov

Would be nice to implement it as a singleton (tooltip shown for only one menu item at a time) and to remove the delay for other items once the tooltip for the first item is opened.

### PR DESCRIPTION
**Pull request by dahaca**
## Description

UI/UX improvements for the whole Suite app, with the exception of onboarding, it can be finished later separately. All major changes approved by Ondra as well. 

The improvements include:

- fixes for obvious visual bugs
- tightening of negative spaces, some places were unnecessarily barren
- Invity section UI
- UI improvement of global components
- transaction history & detail UI
- account list UI
- various modals in settings
- hover effects, tried to add them to everything interactable 
- #5162
- #4111
- https://github.com/trezor/trezor-suite/issues/4067

## Related Issues

Resolve https://github.com/trezor/trezor-suite/issues/5162 
Resolve https://github.com/trezor/trezor-suite/issues/4111 
Resolve https://github.com/trezor/trezor-suite/issues/4067

## Screenshots:

Screenshots for most of the major changes can be found [here in Notion](https://www.notion.so/satoshilabs/UI-Polish-diff-14c53b2676da4f3a8e50ff0a9b7fa551). Because of the amount of changes and some of them being animation-related I don't think it is reasonable to show all diffs.

### CODE REVIEW

File: packages/suite/src/components/firmware/FirmwareInstallation.tsx [hunk 1]
```diff
@@ -76,7 +76,11 @@ export const FirmwareInstallation = ({
         <>
             {(status === 'unplug' || status === 'reconnect-in-normal') && (
                 // Modal to instruct user to disconnect the device and reconnect in normal mode
-                <ReconnectDevicePrompt expectedDevice={cachedDevice} requestedMode="normal" />
+                <ReconnectDevicePrompt
+                    expectedDevice={cachedDevice}
+                    requestedMode="normal"
+                    onClose={onClose}
```
Comment (tomasklim) [inline packages/suite/src/components/firmware/FirmwareInstallation.tsx:84]: I am trying this in Chrome with WebUSB and it "freezes" my app.

`Settings - Device` -> `Update Available` -> `Install firmware` -> `Abort` 

Now I cannot close modal and device is still in `Do you want to restart device in bootloader mode?` state

File: packages/components/src/components/Modal/index.tsx [hunk 2]
```diff
@@ -110,8 +110,9 @@ const Body = styled.div`
 const Description = styled.div`
     color: ${({ theme }) => theme.TYPE_LIGHT_GREY};
     font-size: ${variables.FONT_SIZE.SMALL};
-    margin: 0 auto 10px auto;
+    margin: 0 auto 18px auto;
     max-width: fit-content; /* makes sure the description does not widen the modal beyond content width when the modal's width is "unset" */
+    text-align: left;
```
Comment (tomasklim) [inline packages/components/src/components/Modal/index.tsx:118]: Screenshot 2022-09-05 at 9 23 04
Screenshot 2022-09-05 at 9 23 44
Some content in modals looks kinda weird now?

File: packages/components/src/components/Modal/index.tsx [hunk 3]
```diff
@@ -139,7 +140,7 @@ const ModalWindow = styled.div`
     transition: all 0.3s;
     max-width: 95%;
     min-width: 305px;
-    width: 720px;
+    width: 680px;
```
Comment (tomasklim) [inline packages/components/src/components/Modal/index.tsx:147]: do we really want to make modal windows narrower?
 
Screenshot 2022-09-05 at 9 25 06

File: packages/components/src/components/Timerange/index.tsx [hunk 4]
```diff
@@ -23,22 +23,23 @@ const StyledTimerange = styled.div`
...
+
     & > * + * {
         margin-left: 10px;
     }
 `;
 
 const Calendar = styled.div`
     width: 345px;
-    padding: 10px 10px 0;
+    padding: 12px 12px 0;
```
Comment (tomasklim) [inline packages/components/src/components/Timerange/index.tsx:46]: Tip: 
better border/shadow colour in case of dark theme?
Screenshot 2022-09-05 at 9 26 30

File: packages/components/src/components/Timerange/index.tsx [hunk 5]
```diff
@@ -172,7 +187,8 @@ const Calendar = styled.div`
     .rdrDayStartPreview,
     .rdrDayInPreview,
     .rdrDayEndPreview {
-        border-color: ${props => props.theme.BG_GREEN};
+        border: none;
+        background: ${({ theme }) => theme.BG_LIGHT_GREEN};
```
Comment (tomasklim) [inline packages/components/src/components/Timerange/index.tsx:195]: Screenshot 2022-09-05 at 9 28 38
Hover on date is almost invisible in dark mode

File: packages/components/src/components/Tooltip/index.tsx [hunk 6]
```diff
@@ -147,8 +147,8 @@ const Tooltip = ({
 
     const animationVariants = useMemo<Variants>(
         () => ({
-            shown: { opacity: 1, transform: 'scale(1) translate(0px, 0px)' },
-            hidden: { opacity: 0, transform: `scale(0.9) ${getTranslateStyle(placement)}` },
+            shown: { opacity: 1, transform: ' translate(0px, 0px)' },
```
Comment (tomasklim) [inline packages/components/src/components/Tooltip/index.tsx]: Why there is space before `translate`? same applies for the line below

File: packages/components/src/components/buttons/Button/index.tsx [hunk 7]
```diff
@@ -12,7 +12,7 @@ const getPadding = (variant: ButtonVariant, hasLabel: boolean) => {
         return '4px 6px';
     }
 
-    return hasLabel ? '9px 12px' : '8px';
+    return hasLabel ? '9px 22px' : '8px';
```
Comment (tomasklim) [inline packages/components/src/components/buttons/Button/index.tsx:15]: Is there a button without label? 😆

File: packages/components/src/components/form/InputStyles.ts [hunk 8]
```diff
@@ -98,7 +98,7 @@ export const baseInputStyle = css<BaseInputProps>`
         css`
             background: ${({ theme }) => theme.BG_GREY};
             box-shadow: none;
-            color: ${({ theme }) => theme.TYPE_DARK_GREY};
+            color: ${({ theme }) => theme.TYPE_LIGHT_GREY};
```
Comment (tomasklim) [inline packages/components/src/components/form/InputStyles.ts:101]: Screenshot 2022-09-05 at 9 33 14
The `Universal` disabled button looks weird

File: packages/suite/src/components/wallet/AccountsMenu/components/AccountGroup/index.tsx [hunk 9]
```diff
@@ -28,6 +28,11 @@ const Header = styled.header<{ onClick?: () => void }>`
     font-weight: ${variables.FONT_WEIGHT.MEDIUM};
     color: ${props => props.theme.TYPE_DARK_GREY};
     border-top: 1px solid ${props => props.theme.STROKE_GREY};
+    transition: padding 0.2s ease-in-out;
```
Comment (tomasklim) [inline packages/suite/src/components/wallet/AccountsMenu/components/AccountGroup/index.tsx]: This looks good in the transactions list, but it looks weird here 😆

File: packages/suite/src/components/firmware/SelectCustomFirmware.tsx [hunk 10]
```diff
@@ -1,72 +1,30 @@
 import React, { useState } from 'react';
```
Comment (tomasklim) [inline packages/suite/src/components/firmware/SelectCustomFirmware.tsx:1]: In my opinion, the text is too small now. This modal looks like a different app.

Screenshot 2022-09-05 at 9 54 41

File: packages/suite/src/components/firmware/SelectCustomFirmware.tsx [hunk 11]
```diff
@@ -1,72 +1,30 @@
...
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    flex-shrink: 0;
+const StyledLink = styled(TrezorLink)`
+    margin-left: 6px;
 `;
 
 const StyledDropZone = styled(DropZone)`
+    width: 220px;
```
Comment (tomasklim) [inline packages/suite/src/components/firmware/SelectCustomFirmware.tsx]: Maybe it would be better to have full width?

File: packages/suite/src/components/guide/GuideSearch.tsx [hunk 12]
```diff
@@ -42,9 +42,13 @@ const PreviewContent = styled.div`
 const StyledInput = styled(Input)`
     && {
         background-color: ${({ theme }) => theme.BG_GREY_ALT};
-        border: none;
         border-radius: 8px;
         height: 40px;
+        border-color: ${props => props.theme.BG_GREY_ALT};
+
+        :focus {
+            border-color: ${({ theme }) => theme.STROKE_GREY};
```
Comment (tomasklim) [inline packages/suite/src/components/guide/GuideSearch.tsx]: Invisible in darkmode

File: packages/suite/src/components/suite/modals/AddAccount/components/EnableNetwork.tsx [hunk 13]
```diff
@@ -71,44 +55,28 @@ export const EnableNetwork = ({
...
-                            isActive={isTestnetVisible}
-                            size={16}
-                            color={theme.TYPE_LIGHT_GREY}
-                            icon="ARROW_DOWN"
+                        </P>
+                    }
+                >
+                    <TestnetCoinsWrapper>
+                        <TestnetCoinsDescription>
+                            <StyledIcon size={12} color={theme.TYPE_LIGHT_GREY} icon="INFO" />
```
Comment (tomasklim) [inline packages/suite/src/components/suite/modals/AddAccount/components/EnableNetwork.tsx]: Screenshot 2022-09-05 at 10 09 50
The icon is so small that it does not look like a question mark

Comment (tomasklim) [reply packages/suite/src/components/suite/modals/AddAccount/components/EnableNetwork.tsx]: Btw would be nice to have this info also in Settings > Crypto

File: packages/suite/src/components/suite/modals/AddAccount/index.tsx [hunk 14]
```diff
@@ -18,7 +18,7 @@ import { AddAccountButton } from './components/AddAccountButton';
 import { useDispatch } from 'react-redux';
 
 const StyledModal = styled(Modal)`
-    min-height: 550px;
+    width: 560px;
```
Comment (tomasklim) [inline packages/suite/src/components/suite/modals/AddAccount/index.tsx:21]: Too narrow

File: packages/suite/src/components/suite/modals/ImportTransaction/index.tsx [hunk 15]
```diff
@@ -7,6 +8,10 @@ import { DelimiterForm } from './components/DelimiterForm';
 import { ExampleCSV } from './components/ExampleCSV';
 import { parseCSV } from '@suite-common/wallet-utils';
 
+const StyledModal = styled(Modal)`
+    width: 600px;
+`;
+
 type Props = {
```
Comment (tomasklim) [inline packages/suite/src/components/suite/modals/ImportTransaction/index.tsx:15]: Screenshot 2022-09-05 at 10 15 09
Could you please fix this for a phone?

**Review by tomasklim — CHANGES_REQUESTED** [review]

File: packages/suite/src/components/firmware/FirmwareInstallation.tsx (hunk 1 above)
Comment (dahaca) [reply packages/suite/src/components/firmware/FirmwareInstallation.tsx:84]: I need to move the rendering conditions directly to the AbortButton component, however it should work with WebUSB. Thoughts? @mroz22 

```
 const isAbortable =
        transport?.type === 'bridge'
            ? versionUtils.isNewerOrEqual(transport?.version as string, '2.0.31')
            : true; // Works via WebUSB
```

File: packages/components/src/components/Modal/index.tsx (hunk 3 above)
Comment (dahaca) [reply packages/components/src/components/Modal/index.tsx:147]: Very often they were too wide and even made narrower locally :( 40px is a small compromise, I think.

File: packages/components/src/components/Modal/index.tsx (hunk 2 above)
Comment (dahaca) [reply packages/components/src/components/Modal/index.tsx:118]: Description should be left-aligned if modal contents are, which they are.

File: packages/components/src/components/buttons/Button/index.tsx (hunk 7 above)
Comment (dahaca) [reply packages/components/src/components/buttons/Button/index.tsx:15]: I suppose when it's loading or something, I had the same question :D

File: packages/suite/src/components/suite/modals/AddAccount/index.tsx (hunk 14 above)
Comment (dahaca) [reply packages/suite/src/components/suite/modals/AddAccount/index.tsx:21]: Not really, it's more tight that way and easier to visually scan the options, since there are 1-3 coins in a row ;)

File: packages/components/src/components/Modal/index.tsx (hunk 3 above)
Comment (dahaca) [reply packages/components/src/components/Modal/index.tsx:147]: The modal on the screenshot has a custom width btw

File: packages/components/src/components/buttons/Button/index.tsx (hunk 7 above)
Comment (dahaca) [reply packages/components/src/components/buttons/Button/index.tsx:15]: Yes, several

File: packages/components/src/components/form/InputStyles.ts (hunk 8 above)
Comment (dahaca) [reply packages/components/src/components/form/InputStyles.ts:101]: Shouldn't be a button for now, I suppose? It is because of this:

```
<Button
    variant="tertiary"
    // icon={revision ? 'EXTERNAL_LINK' : undefined}
    // alignIcon="right"
    disabled // TODO: this should link to an article in knowledge base or guide in the future
>
```

File: packages/components/src/components/Timerange/index.tsx (hunk 4 above)
Comment (dahaca) [reply packages/components/src/components/Timerange/index.tsx:46]: done 0d1863e9481818b44342062288dd4c3dc1927b87

File: packages/components/src/components/Tooltip/index.tsx (hunk 6 above)
Comment (dahaca) [reply packages/components/src/components/Tooltip/index.tsx]: done 0d1863e9481818b44342062288dd4c3dc1927b87

File: packages/suite/src/components/firmware/SelectCustomFirmware.tsx (hunk 10 above)
Comment (dahaca) [reply packages/suite/src/components/firmware/SelectCustomFirmware.tsx:1]: I think I will make the modal narrower :D

File: packages/components/src/components/Modal/index.tsx (hunk 3 above)
Comment (tomasklim) [reply packages/components/src/components/Modal/index.tsx:147]: Okay, then this custom width is too narrow

File: packages/suite/src/components/firmware/FirmwareInstallation.tsx (hunk 1 above)
Comment (matejkriz) [reply packages/suite/src/components/firmware/FirmwareInstallation.tsx:84]: same behaviour for me with T1, regardless of use of bridge or webusb

File: packages/suite/src/components/wallet/AccountsMenu/components/AccountGroup/index.tsx [hunk 16]
```diff
@@ -76,16 +81,15 @@ export default forwardRef((props: Props, _ref: React.Ref<HTMLDivElement>) => {
...
-                            <Icon
-                                data-test="@account-menu/arrow"
-                                canAnimate={animatedIcon}
-                                isActive={isOpened}
-                                size={16}
-                                color={theme.TYPE_LIGHT_GREY}
-                                icon="ARROW_DOWN"
-                            />
-                        )}
+
```
Comment (matejkriz) [inline packages/suite/src/components/wallet/AccountsMenu/components/AccountGroup/index.tsx:92]: Why did you remove the condition? Isn't it confusing especially with the hover effect, if it does nothing (it's not possible to close the group)?

File: packages/suite/src/components/wallet/TransactionItem/components/TransactionHeading.tsx [hunk 17]
```diff
@@ -28,11 +28,11 @@ const HeadingWrapper = styled.div`
 
 const ChevronIconWrapper = styled.div<{ show: boolean; animate: boolean }>`
     display: flex;
-    visibility: ${({ show }) => (show ? 'visible' : 'hidden')};
-    margin-left: ${({ animate }) => (animate ? '5px' : '2px')};
+    /* visibility: ${({ show }) => (show ? 'visible' : 'hidden')}; */
```
Comment (matejkriz) [inline packages/suite/src/components/wallet/TransactionItem/components/TransactionHeading.tsx]: remove instead of commenting out?

File: packages/suite/src/components/firmware/FirmwareInstallation.tsx (hunk 1 above)
Comment (dahaca) [reply packages/suite/src/components/firmware/FirmwareInstallation.tsx:84]: done, didn't have the problem


cd71fd0bc6ee3049ff8ff43d3ca6eac28abc10fb

File: packages/components/src/components/Timerange/index.tsx (hunk 5 above)
Comment (dahaca) [reply packages/components/src/components/Timerange/index.tsx:195]: e51360010aedf6346a20ca007d461f2ae80dfe7b

File: packages/suite/src/components/firmware/SelectCustomFirmware.tsx (hunk 11 above)
Comment (dahaca) [reply packages/suite/src/components/firmware/SelectCustomFirmware.tsx]: [dad90c9](https://github.com/trezor/trezor-suite/pull/6168/commits/dad90c9f640f4efd44175ba250ecfb8088ab2a92)

File: packages/suite/src/components/firmware/SelectCustomFirmware.tsx (hunk 10 above)
Comment (dahaca) [reply packages/suite/src/components/firmware/SelectCustomFirmware.tsx:1]: [dad90c9](https://github.com/trezor/trezor-suite/pull/6168/commits/dad90c9f640f4efd44175ba250ecfb8088ab2a92)

File: packages/suite/src/components/suite/modals/AddAccount/components/EnableNetwork.tsx (hunk 13 above)
Comment (dahaca) [reply packages/suite/src/components/suite/modals/AddAccount/components/EnableNetwork.tsx]: [dad90c9](https://github.com/trezor/trezor-suite/pull/6168/commits/dad90c9f640f4efd44175ba250ecfb8088ab2a92)

File: packages/components/src/components/form/InputStyles.ts (hunk 8 above)
Comment (dahaca) [reply packages/components/src/components/form/InputStyles.ts:101]: [dad90c9](https://github.com/trezor/trezor-suite/pull/6168/commits/dad90c9f640f4efd44175ba250ecfb8088ab2a92)

File: packages/suite/src/components/wallet/TransactionItem/components/TransactionHeading.tsx (hunk 17 above)
Comment (dahaca) [reply packages/suite/src/components/wallet/TransactionItem/components/TransactionHeading.tsx]: [2259879](https://github.com/trezor/trezor-suite/pull/6168/commits/22598792797b8c4022b75724baf2ef5f6bef4842)

File: packages/suite/src/components/wallet/AccountsMenu/components/AccountGroup/index.tsx (hunk 16 above)
Comment (dahaca) [reply packages/suite/src/components/wallet/AccountsMenu/components/AccountGroup/index.tsx:92]: Because it's better to see the `expanded` state, with the icon upside down, than to remove the icon completely. We don't remove it in Select :D And I changed the hover effect. 

[2259879](https://github.com/trezor/trezor-suite/pull/6168/commits/22598792797b8c4022b75724baf2ef5f6bef4842)

File: packages/suite/src/components/suite/modals/ImportTransaction/index.tsx (hunk 15 above)
Comment (dahaca) [reply packages/suite/src/components/suite/modals/ImportTransaction/index.tsx:15]: [2259879](https://github.com/trezor/trezor-suite/pull/6168/commits/22598792797b8c4022b75724baf2ef5f6bef4842)

File: packages/suite/src/components/wallet/AccountsMenu/components/AccountGroup/index.tsx (hunk 9 above)
Comment (dahaca) [reply packages/suite/src/components/wallet/AccountsMenu/components/AccountGroup/index.tsx]: [2259879](https://github.com/trezor/trezor-suite/pull/6168/commits/22598792797b8c4022b75724baf2ef5f6bef4842)

File: packages/suite/src/components/guide/GuideSearch.tsx (hunk 12 above)
Comment (dahaca) [reply packages/suite/src/components/guide/GuideSearch.tsx]: [536346a](https://github.com/trezor/trezor-suite/pull/6168/commits/536346ac0d0d6adcdd9e7ed0120ca6df31dbaa18)

File: packages/suite/src/components/firmware/FirmwareInstallation.tsx (hunk 1 above)
Comment (tomasklim) [reply packages/suite/src/components/firmware/FirmwareInstallation.tsx:84]: still does not work

File: packages/suite/src/components/suite/modals/AddAccount/components/EnableNetwork.tsx (hunk 13 above)
Comment (tomasklim) [reply packages/suite/src/components/suite/modals/AddAccount/components/EnableNetwork.tsx]: It is now in Settings > Crypto. However, the icon on the non-retina display is still blurred

**Comment by tomasklim** [comment]
Could you please squash and rebase before the next review round? 🫣

File: packages/suite/src/components/firmware/FirmwareInstallation.tsx (hunk 1 above)
Comment (tomasklim) [reply packages/suite/src/components/firmware/FirmwareInstallation.tsx:84]: It disappeared for Trezor 1, still there for Trezor T. It should probably be like that right?

I see one more problem. If you hit `ESC` here on Model 1. It breaks the app.

Screenshot 2022-09-09 at 10 00 21

https://user-images.githubusercontent.com/33235762/189301932-e3c28e3a-3971-4f75-a948-d1a101f8f37c.mov

**Review by tomasklim — CHANGES_REQUESTED** [review]
Icon colors are totally messed up 

Screenshot 2022-09-09 at 10 08 33
Screenshot 2022-09-09 at 10 08 46

Screenshot 2022-09-09 at 10 09 28
Screensh
ot 2022-09-09 at 10 09 05

File: packages/components/src/components/form/Select/index.tsx [hunk 18]
```diff
@@ -11,7 +11,7 @@ import ReactSelect, {
     SelectInstance,
```
Comment (tomasklim) [inline packages/components/src/components/form/Select/index.tsx]: Probably not related to changes in this component, but it is select
Screenshot 2022-09-09 at 10 19 10

**Review by tomasklim — CHANGES_REQUESTED** [review]

File: packages/components/src/components/Timerange/index.tsx (hunk 4 above)
Comment (tomasklim) [reply packages/components/src/components/Timerange/index.tsx:46]: @dahaca can you also add it here, please? 
Screenshot 2022-09-09 at 10 23 36

File: packages/suite/src/components/firmware/FirmwareInstallation.tsx (hunk 1 above)
Comment (dahaca) [reply packages/suite/src/components/firmware/FirmwareInstallation.tsx:84]: I don't remember if I told you or not but on the step when you are required to press yes/no on T1 the button won't be present. If you press NO on T1 you will see the same screen as on TT, prompting to activate the bootloader mode **manually**. In the past, the only way to quit this dialogue was pressing ESC which is not so obvious or reloading the page. 

The bug with ESC was always there apparently... 😨 I made it so the modal is not cancelable during that step but you still can use the abort button. Meaning that during the phase where you wait for confirmation for automatic bootloader mode on T1 there is no other way to proceed other than pressing NO on the device.

ee8bdcf4ffb9af1582b1031815fc52b944764303

File: packages/components/src/components/form/Select/index.tsx (hunk 18 above)
Comment (dahaca) [reply packages/components/src/components/form/Select/index.tsx]: 266e3024f0bfac65f1a20cf0be353499d9d8642e

**Comment by dahaca** [comment]
> Icon colors are totally messed up

266e3024f0bfac65f1a20cf0be353499d9d8642e

File: packages/suite/src/components/suite/modals/AddAccount/components/EnableNetwork.tsx [hunk 19]
```diff
@@ -1,50 +1,21 @@
-import React, { useState } from 'react';
+import React from 'react';
```
Comment (tomasklim) [inline packages/suite/src/components/suite/modals/AddAccount/components/EnableNetwork.tsx:1]: One more thing, it should be green when enabled.

https://user-images.githubusercontent.com/33235762/189847420-5608baf0-21c7-4dad-bc63-f7e9b22c6aa4.mov

**Review by tomasklim — APPROVED** [review]
Very nice, beautiful, fakt.

Please fix coin border and let's merge it!

**Comment by dahaca** [comment]
FOR QA:

Make sure nothing in the app looks broken. The onboarding section hasn't been touched. 

Except for https://github.com/trezor/trezor-suite/issues/4111, this PR included no logical changes, everything is visual.

**Comment by bosomt** [comment]
@dahaca was dry run modals part of UI uplift ? I noted that it looks different at beginning.
When I disconnect device text is misaligned .

image

**Info:**
 - Suite version: web 22.10.0 (22d82a98b0b62a463567c20f3fa21b94e4c0495c)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:104.0) Gecko/20100101 Firefox/104.0
 - OS: MacIntel
 - Screen: 1848x1155
 - Device: model T 2.5.2 Universal (revision 0d87b55ba4fed7eecc72bf2a94ee473830b095e9)
 - Transport: bridge 2.0.32

**Comment by bosomt** [comment]
**QA OK**

**Info:**
 - Suite version: desktop 22.10.0 (07a9a0d73a0f1482e8d466cc8c5ee6ed410098ed)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) TrezorSuite/22.10.0 Chrome/100.0.4896.160 Electron/18.3.5 Safari/537.36
 - OS: MacIntel
 - Screen: 1680x1050
 - Device: model T 2.5.3 Universal (revision 9cec8f057a28afb2f6d432cc90c0a586833b6670)
 - Transport: bridge 2.0.32
